src/assets/scripts/client/InputController.js

```diff
@@ -450,9 +450,11 @@ export default class InputController {
             case KEY_CODES.ESCAPE: {
                 this.closeAllDialogs();
 
-                if (!_includes(currentCommandInputValue, this.input.callsign) ||
-                    currentCommandInputValue.trim() === this.input.callsign
-                ) {
+                const noCallsign = !_includes(currentCommandInputValue, this.input.callsign);
+                const hasOnlyCallsign = currentCommandInputValue.trim() === this.input.callsign;
+                const noSelectedCallsign = this.input.callsign === "";
+
+                if (noCallsign || hasOnlyCallsign || noSelectedCallsign) {
```

Comment:
I'd like for you to rename two of these _(also note the single quotes)_:
```js
const hasCallsign = _includes(currentCommandInputValue, this.input.callsign);
const hasSelectedCallsign = this.input.callsign === '';
```

Then in your if, just reverse the logic for those two thusly:
```js
 if (!hasCallsign || hasOnlyCallsign || !hasSelectedCallsign) {
  // ...
}
```

Reply:
Done by 61a95aa e38f014